tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
 
 
 // =========================
 // Start
 // =========================
-//x @step(r'I start the possibly error-producing interview "([^"]+)"')
+//x @step(r'I start the possibly error-producing interview "([^"]+)"') 
 //x @step(r'I am using the server "([^"]+)"')
 //x @step(r'I log in with "([^"]+)" and "([^"]+)"')
-//x @step(r'I launch the interview "([^"]+)"')
+//x @step(r'I launch the interview "([^"]+)"') 
 
 // ? @step(r'I start the interview "([^"]+)"')
 // ? I start the interview file "([^"]+)" on mobile
-// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???)
+// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???) 
 // ? I open the interview file ?"?([^"]+)"?[ on ]?(.*)/
-// √ I start the interview on mobile|pc
+// √ I start the interview on mobile|pc 
 
 
 // =========================
 // Observational/Passive
 // =========================
-//x @step(r'I wait forever')
+//x @step(r'I wait forever') 
 //x @step(r'I should see "([^"]+)" as the title of the page')
 //x @step(r'I should see "([^"]+)" as the URL of the page')
 //x @step(r'I save a screenshot to "([^"]+)"')
-//x @step(r'I set the window size to ([0-9]+)x([0-9]+)')
+//x @step(r'I set the window size to ([0-9]+)x([0-9]+)') 
 //x @step(r'I screenshot the page')
 
 // √ @step(r'I wait (\d+) seconds?')
 
-// Make writing tests more comfortable.
+// Make writing tests more comfortable. 
 // √ I do nothing
 
 // ~ I should see the term "([^"]+)"
 
-// √ /I (should|should not) see the phrase "([^"]+)"/i
+// √ /I (should|should not) see the phrase "([^"]+)"/i 
 
 // √ For things that have double quotes in them
-// √ I (should|should not) see the phrase '([^']+)'/i
+// √ I (should|should not) see the phrase '([^']+)'/i 
 
-// √ the question id should be {string}
+// √ the question id should be {string} 
 
 // √ I will be told an answer is invalid
 
-// ? @step(r'I should see that "([^"]+)" is "([^"]+)"')  // What do? Anything with label
+// ? @step(r'I should see that "([^"]+)" is "([^"]+)"')  // What do? Anything with label 
 
 // ? "" is selected/picked?
 // √ the third "blah" checkbox in "bleh" is checked
 
-// √ I arrive at the next page
+// √ I arrive at the next page 
 
 // √ I (don't|can't) continue
 
-// √ the link "([^"]+)" should lead to "([^"]+)"
+// √ the link "([^"]+)" should lead to "([^"]+)" 
 
 // √ the link "([^"]+)" should open in (a new window|the same window)
 
 
 // =========================
 // Navigational
-// =========================
+// ========================= 
 //x @step(r'I exit by taping "([^"]+)"')
 //x @step(r'If I see it, I will tap the link "([^"]+)"')
 
-// ? I tap the ${ specified } link "([^"]+)"
+// ? I tap the ${ specified } link "([^"]+)" 
 
 // Also sometimes not navigational
-// ^ @step(r'I tap the button "([^"]+)"')
+// ^ @step(r'I tap the button "([^"]+)"') 
 // ? I tap "([^"]+)" (can this be a button or link or anything?) (Include 1st, 2nd, etc?)
```

Comment:
Another pretty broad sentence, similar to `I pick ""`

Reply:
Yeah, I suppose it would be a sentence that could be used for buttons, links, radio button choices, and other fields. I'll make it an 'x'.